docs/Agent/user-guide/man_page.md

```diff
@@ -1,13 +1,33 @@
 # Man pages
+ 
+**Contents**
+
+#### Performance tool management commands
```

Comment:
Since `Man pages` above is a 1st level header, why are we specifying the TOC and other sections as 4th level?

I'm not really sure that the TOC headers should be sections, although it might be convenient to be able to link to them; but pushing everything down to `####` level seems excessive!

Reply:
I'm not sure what "looks OK" has to do with the logical structure of the page. It'd make more sense to have header level 1, 2, 3 instead of starting at 1 and then skipping to 4 because it "looks OK". Sigh.